algobot/__main__.py

```diff
@@ -321,24 +435,39 @@ def initiate_optimizer(self):
             return
 
         combos = self.configuration.get_optimizer_settings()
-        if combos['strategies'] == {}:
-            create_popup(self, "No strategies found. Make sure you have some strategies for optimization.")
+        if combos["strategies"] == {}:
+            create_popup(
+                self,
```

Comment:
Yeah, I don't think this is honestly that readable... No need to separate in into 4 separate lines imo